src/js/components/helptips/onboardingcompletetip.js

```diff
@@ -75,14 +75,13 @@ export class OnboardingCompleteTip extends React.Component {
             might have to refresh the page.
           </p>
           <p>You&apos;ve now got a good foundation in how to use Mender. Look for more help hints in the UI as you go along.</p>
-          <p>{`If you used one of our pre-built images you can start using full-image${hasDeltaAccess ? ` and delta` : ''} updates right away.`}</p>
           What next?
           <div>
-            <a href={`https://docs.mender.io/${docsVersion}artifacts/snapshots`} target="_blank">
+            <a href={`https://docs.mender.io/${docsVersion}get-started/deploy-a-system-update`} target="_blank">
               Learn about full-image updates
             </a>{' '}
             or{' '}
-            <a href="https://hub.mender.io/c/update-modules" target="_blank">
+            <a href={`https://docs.mender.io/${docsVersion}get-started/deploy-a-container-update`} target="_blank">
               how to create other kinds of application updates.
```

Comment:
I think the text at the end should be more explicit as we are now linking to tutorials and not reference text.

Suggestion:

```
What next?

Proceed to one of the following tutorials (listed in recommended order):

1. Deploy a system update
2. Deploy a container update
```